tic-tac-toe-app/styles.css

```diff
@@ -0,0 +1,32 @@
+html {
+    font-family: monospace;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+}
+
+.gameCell {
+    height: 130px;
+    width: 130px;
+    border: 2px solid black;
+}
+
+.board {
```

Comment:
At certain screen sizes, the board's configuration is no longer 3x3. 

![image](https://user-images.githubusercontent.com/38898288/80898657-fe42cc00-8cd3-11ea-9dc1-02d0599765f4.png)

It definitely looks cool! Consider updating this style, though, so that users can have a consistent gaming experience regardless of screen size.